internal/corazarules/rule_match.go

```diff
@@ -229,6 +231,22 @@ func (mr MatchedRule) ErrorLog() string {
 		}
 	}
 
-	log.WriteString("\n")
```

Comment:
Removing this line should fix https://github.com/corazawaf/coraza-caddy/issues/87 and supersede https://github.com/corazawaf/coraza-caddy/pull/93. Now writing the new line is delegated to the ErrorLog() caller.